database/seeds/DemoActivityIdsAssociationSeeder.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class DemoActivityIdsAssociationSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        return DB::transaction(function () {
+
+            $sampleProjectName = config('constants.curriki-sample-project');
+            $demoUserEmail = config('constants.curriki-demo-user-email');
+            $organizationId = DB::table('organizations')->where('domain', 'currikistudio')->value('id');
+dd( $sampleProjectName );
```

Comment:
There is dd( $sampleProjectName ) in database/seeds/DemoActivityIdsAssociationSeeder.php;
Please remove it.